tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml

```diff
@@ -259,21 +259,23 @@
     fail_msg: "Unable to detach VM from VG"
     success_msg: "VM detached from VG successfully"
 
-- name: Fetch all VMs attached to VG
-  ntnx_volume_groups_vms_info_v2:
-    ext_id: "{{ vg1_uuid }}"
-  register: result
+# Need to verify here that VM is detached from VG
```

Comment:
Use VG info to check if VM is attached or dettached. This is critical verification dont remove it

Reply:
VG info doesnt have attachments info. Ignore. Remove testcase

Reply:
Removed